buildkite/src/Command/MinaArtifact.dhall

```diff
@@ -98,7 +98,24 @@ let hardforkPipeline : DebianVersions.DebVersion -> Pipeline.Config.Type =
         , mode = PipelineMode.Type.PackageGeneration
         }
       , steps =
-        [ Command.build
+        [ Command.build Command.Config::{
+            commands = [
+              Cmd.runInDocker Cmd.Docker::{
+                image = "\\\${MINA_DAEMON_DOCKER_IMAGE}"
```

Comment:
```suggestion
                image = "\\\${MINA_DAEMON_DOCKER_IMAGE:-gcr.io/o1labs-192920/mina-daemon@sha256:6da66879aacab050a6955c84347f587e43044987b04b9de4522a942f770cc5e7}"
```

I took this image from [recent HF package nightly build](https://buildkite.com/o-1-labs-2/hardfork-package-generation/builds/270#018f32be-4103-4ae4-9a2a-24c7f11c0b7d). Devnet/mainnet distinction doesn't affect `mina-create-genesis` tool.